src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/UserProvidedInstanceTypeList/UserProvidedInstanceTypeList.tsx

```diff
@@ -51,29 +62,29 @@ const UsersInstanceTypesList: FC<UsersInstanceTypesListProps> = ({ userInstanceT
   };
 
   const filteredItems = useMemo(() => {
-    return userInstanceTypes.filter(
+    return userProvidedInstanceTypes.filter(
       (opt) =>
         isEmpty(searchInput) ||
         getName(opt).toLowerCase().includes(searchInput.toString().toLowerCase()),
     );
-  }, [searchInput, userInstanceTypes]);
+  }, [searchInput, userProvidedInstanceTypes]);
 
   const { columns, getSortType, sortedData } = useInstanceTypeListColumns(
     filteredItems,
     pagination,
   );
 
-  if (activeNamespace === ALL_NAMESPACES_SESSION_KEY) {
+  if (isAllNamespaces(activeNamespace) && noUserCreatedClusterITsExist(userProvidedInstanceTypes)) {
```

Comment:
instead of noUser... can !isEmpty(userProvidedInstanceTypes) work? as if its all projects, only cluster IT will be in userProvidedInstanceTypes...

Reply:
Yup. We can do that. I forgot that `useVirtualMachineInstanceTypes` updates based on the active namespace. Thanks for catching that!